server/helpers/middlewares/video-abuses.ts

```diff
@@ -1,9 +1,17 @@
 import { Response } from 'express'
 import { VideoAbuseModel } from '../../models/video/video-abuse'
+import { fetchVideo } from '../video'
 
-async function doesVideoAbuseExist (abuseIdArg: number | string, videoId: number, res: Response) {
+async function doesVideoAbuseExist (abuseIdArg: number | string, videoUUID: string, res: Response) {
```

Comment:
It breaks the API no?

Reply:
Yes, but I made sure the rest of the API isnt used in our code.